arts-university-bournemouth.csl

```diff
@@ -12,13 +12,13 @@
     </author>
     <category citation-format="author-date"/>
     <category field="generic-base"/>
-    <updated>2017-04-24T10:20:30+00:00</updated>
+    <updated>2017-08-16T12:39:28+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <macro name="editor">
     <names variable="editor" delimiter=", ">
       <name and="text" initialize-with=". " delimiter=", " sort-separator=", " name-as-sort-order="all"/>
-      <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
+      <label form="short" prefix=" (" suffix=")." strip-periods="true"/>
```

Comment:
@Dbelk, another minor nitpick, but it's better to delete both the period and the `strip-periods="true"` here, which should give the same output.